LEAF_Request_Portal/templates/print_subindicators_ajax.tpl

```diff
@@ -228,7 +228,7 @@
             <!--{if trim($indicator.value) != ''}-->
             <div style="padding: 0px">
                 <!--{if $indicator.displayedValue != ''}-->
-                    <a href="<!--{$orgchartPath}-->/?a=view_employee&empUID=<!--{$indicator.value|escape}-->"><!--{$indicator.displayedValue|sanitize}--></a>
```

Comment:
Any reason why this link can't be retained? This represents a regression in functionality.

Side note: An e2e test for something like this would help highlight potential regressions. @Pelentan

Reply:
The concern for this was that the link is based on the data value / empUID.  If the number is recycled the link would go to an incorrect page.  It had been mentioned and the thinking at that time was that it should be avoided for that reason.